code/modules/jobs/job_types/chaplain.dm

```diff
@@ -4,7 +4,7 @@
 	department_head = list("Head of Personnel")
 	department_flag = CIVILIAN
 	faction = "Station"
-	total_positions = 1
+	total_positions = 3
```

Comment:
have you considered the impact on cult gamemode?

Reply:
yes imagine 10+ round start cultists